pkg/noun/nock.h

```diff
@@ -4,12 +4,17 @@
 #define U3_NOCK_H
 
 #include <stdio.h>
+#include <fcntl.h>
+#include <unistd.h>
 
 #include "c3/c3.h"
 #include "jets.h"
 #include "types.h"
 #include "zave.h"
 
+#define SLOW_STACK_NAME  "/spin_stack_page"
```

Comment:
This actually might get shared with every urbit running on the host. we want to have some distinguisher (pier name?) in this to help

Reply:
This now uses the pid (for king) and ppid (for serf) to uniquely identify each pier uniquely